Makefile

```diff
@@ -18,19 +18,21 @@ endif
 
 DOCKER_ORG ?= falcosecurity
 
-IMAGE_NAME_BUILDER ?= docker.io/$(DOCKER_ORG)/driverkit-builder
+BUILDERS := $(patsubst build/builder_%.Dockerfile,%,$(wildcard build/builder_*.Dockerfile))
```

Comment:
List all builders name.
NOTE that builder image name should always be:
`builder_$osname.Dockerfile`